cmake/onnxruntime_mlas.cmake

```diff
@@ -228,8 +241,8 @@ else()
         set_source_files_properties(${mlas_platform_srcs_avx512bw} PROPERTIES COMPILE_FLAGS "-mavx512bw")
       endif()
     else()
-      # Disable platform support for AVX512.
-      set_source_files_properties(${mlas_common_srcs} PROPERTIES COMPILE_FLAGS "-DMLAS_AVX512_UNSUPPORTED")
+      # Do not compile CPP support for 512BW
```

Comment:
Same here, spell out AVX512BW. I preferred the period at the end of the sentence too.